### PR TITLE
Remove all spaces in the end of lines

### PR DESCRIPTION
**Pull request by Ivanidzo4ka**
It's nice to not have spaces in the end of lines

### CODE REVIEW

**Comment by Ivanidzo4ka** [comment]
@zeahmed  @sfilipi  Robotom is really unhappy with spaces in the end of line, so if you don't mind I would pacify it :)

File: src/Microsoft.ML.FastTree/FastTreeTweedie.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public sealed partial class FastTreeTweedieTrainer : BoostingFastTreeTrainerBase
         public const string LoadNameValue = "FastTreeTweedieRegression";
         public const string UserNameValue = "FastTree (Boosted Trees) Tweedie Regression";
         public const string Summary = "Trains gradient boosted decision trees to fit target values using a Tweedie loss function. This learner is a generalization of Poisson, compound Poisson, and gamma regression.";
-        new public const string Remarks = @"<remarks>
+        public new const string Remarks = @"<remarks>
```
Comment (zeahmed) [inline src/Microsoft.ML.FastTree/FastTreeTweedie.cs]: Following the [documentation](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2010/435f1dw2(v=vs.100)), I see that keyword `new` is the first in statements.

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.FastTree/FastTreeTweedie.cs]: Let me add real @TomFinley 
RoboTom gives this:
warning TLC_ExplicitAccessModifiers: Symbol 'Remarks' did not have access modifiers as the leading modifiers

Comment (TomFinley) [reply src/Microsoft.ML.FastTree/FastTreeTweedie.cs]: The cold robotic hand of robotom was not programmed for this eventuality, due to the limitations of its flawed, human, designer. I might prefer to change robotom vs. do this.

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.FastTree/FastTreeTweedie.cs]: reverted. thank you for bringing this up Zeeshan!

---
In reply to: [199651709](https://github.com/dotnet/machinelearning/pull/478#discussion_r199651709) [](ancestors = 199651709)

File: src/Microsoft.ML.StandardLearners/Standard/PoissonRegression/PoissonRegression.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@ public sealed class PoissonRegression : LbfgsTrainerBase<Float, PoissonRegressio
         internal const string UserNameValue = "Poisson Regression";
         internal const string ShortName = "PR";
         internal const string Summary = "Poisson Regression assumes the unknown function, denoted Y has a Poisson distribution.";
-        new internal const string Remarks = @"<remarks>
+        internal new const string Remarks = @"<remarks>
```
Comment (zeahmed) [inline src/Microsoft.ML.StandardLearners/Standard/PoissonRegression/PoissonRegression.cs]: @Ivanidzo4ka, It has be reverted here as well...:)
I have opened PR (486) to fix it.

**Pull request merged**